app/views/pages/home.html.erb

```diff
@@ -1,3 +1,13 @@
+
+
+<%= render "shared/banner"%>
```

Comment:
Please remove the empty spaces at the top and add a space between "shared/banner" and %>